### DISCUSSION TITLE
pre-trained model

### DISCUSSION THREAD

**Discussion by WJZYN** [rasbt/deeplearning-models#49] [Q&A]
thanks a lot about an amazing project. I want to ask if there are pretrained model of Resnet18 trained on celeba.

**Comment by rasbt** [comment]
Glad to hear you find it useful. Unfortunately, I don't have any pre-trained models due to the large file sizes. Also, I am not sure what task you are looking for. However, here is a Resnet-18 trained on the CelebA Gender attribute. It trains in ca 40 min: https://github.com/rasbt/deeplearning-models/blob/master/pytorch_ipynb/cnn/cnn-resnet18-celeba-dataparallel.ipynb
